src/app/actions/i18n.ts

```diff
@@ -0,0 +1,10 @@
+import { BaseAction } from '.';
+
+export const I18N_READY = 'EXTENSION/I18N_READY';
+export interface I18nReady extends BaseAction {
+  type: typeof I18N_READY;
+}
+
+export const i18nReady = (): I18nReady => ({
```

Comment:
what does this action do?

Reply:
in fact it does not do anything right now!

Reply:
I mean we dispatch it but we don't do anything else with it